autoload/clap/installer.vim

```diff
@@ -102,6 +106,7 @@ function! clap#installer#download_binary() abort
   if has('win32')
     let cmd = 'Powershell.exe -ExecutionPolicy ByPass -File "'.s:plugin_root_dir.'\install.ps1"'
   else
+    call s:unix_sanity_check()
```

Comment:
It could better to just stop the following execution if the check fails:

```vim
if !s:unix_sanity_check_is_ok()
  return
endif
```

Reply:
Thought that `clap#helper#echo_error` aborts the call :) Just pushed a new commit.